VariableAnalysis/Tests/CodeAnalysis/VariableAnalysisTest.php

```diff
@@ -140,6 +140,15 @@ public function testClassWithMembersErrors() {
     $this->assertEquals($expectedErrors, $lines);
   }
 
+  public function testTraitWithMembersErrors() {
+    $fixtureFile = $this->getFixture('TraitWithMembersFixture.php');
+    $phpcsFile = $this->prepareLocalFileForSniffs($this->getSniffFiles(), $fixtureFile);
+    $phpcsFile->process();
+    $lines = $this->getErrorLineNumbersFromFile($phpcsFile);
```

Comment:
Can this be changed to `getWarningLineNumbersFromFile()` (maybe change the test name too) so it will pick up line 64 in your fixture?

Reply:
Sure, last commit contains the requested change. Renamed the function from `testTraitWithMembersErrors` to `testTraitWithMembersWarnings` as well